modules/KalturaSupport/components/dualScreen/imagePlayer.js

```diff
@@ -104,6 +106,28 @@
 		addBinding: function(){
 			var _this = this;
 
+			this.bind('KalturaSupport_ThumbCuePointsReady' , function(){
+				if (_this.getPlayer().isLive()){
+					//Forcing 1st slide is only on VOD entry
+					return;
+				}
+				// on non-autoplayed player - check if required to force-show the 1st CP
+				var chaptersRawData = _this.getCuePoints();
+				//Since imagePlayer doesn't have a uiconf element - copy the value from dualScreen which has representation in the uiconf;
+				if (_this.embedPlayer.plugins.hasOwnProperty("chapters")
+					&& _this.embedPlayer.getKalturaConfig("chapters","showFirstSlideOnLoad")){
```

Comment:
@OrenMe maybe this line 116 can explain it ;)
I do see a use-case where there is no dualScreen plugin and only chapters so I thought Chapters is better 
what do you think?